malaria-ios/ViewControllers/MythVsFactViewController.swift

```diff
@@ -0,0 +1,83 @@
+import UIKit
+
+class MythVsFactViewController: GameViewController {
+  
+  @IBOutlet weak var statementNumberLabel: UILabel!
+  @IBOutlet weak var statementLabel: UILabel!
+  
+  @IBOutlet weak var yesView: UIImageView!
+  @IBOutlet weak var noView: UIImageView!
+  
+  var MVFGame: MythVsFactGame!
+  
+  var statementLabelInitialFrame: CGRect!
+  
+  override var currentLevel: Int {
+    didSet {
+      
+      // Check if we ran out of questions
+      if currentLevel >= MVFGame!.entries.count {
+        endGame()
+        return
+      }
+      
+      // Set the statement
+      statementLabel.text = MVFGame?.entries[currentLevel].statement
+      statementNumberLabel.text = "Statement \(currentLevel + 1) / \(MVFGame!.entries.count)"
+    }
+  }
+  
+  override func viewWillAppear(animated: Bool) {
+    
+    MVFGame = game as! MythVsFactGame
```

Comment:
Btw the class Game at this moment should be a protocol because it is abstract class with only fields and no methods.